src/utils/toasts.js

```diff
@@ -38,10 +37,7 @@ export const toastWalletBackup = (isWalletBackedUp: boolean) => {
     title: 'Please ensure you backup your wallet now',
     autoClose: false,
     onPress: () => {
-      const action = NavigationActions.navigate({
-        routeName: SETTINGS,
-        params: { scrollTo: KEY_SECTION },
-      });
+      const action = NavigationActions.navigate(RECOVERY_SETTINGS);
```

Comment:
Pressing on this toast leads nowhere now ;/

```suggestion
      const action = NavigationActions.navigate({
        routeName: RECOVERY_SETTINGS,
      });
```